app_spec/test/test.js

```diff
@@ -79,27 +79,56 @@ scenario1.runTape('create_post', async (t, { alice }) => {
   t.equal(result.Ok, "QmY6MfiuhHnQ1kg7RwNZJNUQhwDxTFL45AAPnpJMNPEoxk")
 })
 
+scenario2.runTape('delete_post', async (t, { alice, bob }) => {
 
-scenario1.runTape('delete_post', async (t, { alice }) => {
-  t.plan(3)
-
-  const content = "Hello Holo world 321"
-  const in_reply_to = null
-  const params = { content, in_reply_to }
-  const createResult = alice.call("blog", "create_post", params)
-
-  t.ok(createResult.Ok)
+  //create post
+ const alice_create_post_result = await alice.callSync("blog", "create_post",
+    { "content": "Posty", "in_reply_to": "" }
+  )
 
-  const deletionParams = { post_address: createResult.Ok }
-  const deletionResult = alice.call("blog", "delete_post", deletionParams)
+  
+  const bob_create_post_result = await alice.callSync("blog", "posts_by_agent",
```

Comment:
This should be
```suggestion
  const bob_create_post_result = await bob.callSync("blog", "posts_by_agent",
```
now since it's even calling `posts_by_agent`, right?

Reply:
yes! Missed that part thank you!